resources/options_lookup.tsv

```diff
@@ -419,11 +419,11 @@ Insulation Floor	Ceiling R-19	ResStockArguments	floor_over_foundation_assembly_r
-Insulation Foundation Wall	"Wall R-10, Exterior"	ResStockArguments	foundation_wall_type=solid concrete	foundation_wall_thickness=auto	foundation_wall_insulation_r=10	foundation_wall_insulation_location=interior	foundation_wall_insulation_distance_to_top=0	foundation_wall_insulation_distance_to_bottom=auto																				
-Insulation Foundation Wall	"Wall R-15, Exterior"	ResStockArguments	foundation_wall_type=solid concrete	foundation_wall_thickness=auto	foundation_wall_insulation_r=15	foundation_wall_insulation_location=interior	foundation_wall_insulation_distance_to_top=0	foundation_wall_insulation_distance_to_bottom=auto																				
-Insulation Foundation Wall	"Wall R-13, Interior"	ResStockArguments	foundation_wall_type=solid concrete	foundation_wall_thickness=auto	foundation_wall_insulation_r=13	foundation_wall_insulation_location=interior	foundation_wall_insulation_distance_to_top=0	foundation_wall_insulation_distance_to_bottom=auto																				
+Insulation Foundation Wall	Uninsulated	ResStockArguments	foundation_wall_type=solid concrete	foundation_wall_thickness=auto	foundation_wall_insulation_r=0	foundation_wall_insulation_location=interior	foundation_wall_insulation_distance_to_top=0	foundation_wall_insulation_distance_to_bottom=0	rim_joist_assembly_r=3.4																			
+Insulation Foundation Wall	"Wall R-5, Exterior"	ResStockArguments	foundation_wall_type=solid concrete	foundation_wall_thickness=auto	foundation_wall_insulation_r=5	foundation_wall_insulation_location=exterior	foundation_wall_insulation_distance_to_top=0	foundation_wall_insulation_distance_to_bottom=auto	rim_joist_assembly_r=7.5																			
+Insulation Foundation Wall	"Wall R-10, Exterior"	ResStockArguments	foundation_wall_type=solid concrete	foundation_wall_thickness=auto	foundation_wall_insulation_r=10	foundation_wall_insulation_location=exterior	foundation_wall_insulation_distance_to_top=0	foundation_wall_insulation_distance_to_bottom=auto	rim_joist_assembly_r=9.8																			
+Insulation Foundation Wall	"Wall R-13, Interior"	ResStockArguments	foundation_wall_type=solid concrete	foundation_wall_thickness=auto	foundation_wall_insulation_r=13	foundation_wall_insulation_location=interior	foundation_wall_insulation_distance_to_top=0	foundation_wall_insulation_distance_to_bottom=auto	rim_joist_assembly_r=11.3																			
+Insulation Foundation Wall	"Wall R-15, Exterior"	ResStockArguments	foundation_wall_type=solid concrete	foundation_wall_thickness=auto	foundation_wall_insulation_r=15	foundation_wall_insulation_location=exterior	foundation_wall_insulation_distance_to_top=0	foundation_wall_insulation_distance_to_bottom=auto	rim_joist_assembly_r=12.1																			

```

Comment:
Where did `rim_joist_assembly_r` come from? If it is continuous exterior insulation shouldn't `rim_joist_assembly_r` be at least as high as the insulation value since there is no thermal breaks from joists? e.g., Wall R-15, Exterior would have `rim_joist_assembly_r` >= 15. 

Then the interior options would probably be similar to wood stud wall insulation due to the joists spaced every ~16 inches.

Reply:
I added argument `rim_joist_insulation_r` to ResStockArguments. Then `rim_joist_assembly_r` is calculated as the sum of `wall_assembly_r` and `rim_joist_insulation_r`.

Reply:
I don't think the `wall_assembly_r` value can be assumed for the rim joist, this would over insulate in most cases. I think we need to use a constant R-value for a 2-by and tack it onto the rim_joist_insulation_r for the exterior insulation options.  

For the R-13 interior insulation, `rim_joist_assembly_r` = 11.3 makes sense, since it should be pretty similar to the R-13 assembly R value for wood stud wall insulation